command/genesis/genesis.go

```diff
@@ -126,6 +128,18 @@ func setFlags(cmd *cobra.Command) {
 			command.DefaultGenesisGasLimit,
 		),
 	)
+	cmd.Flags().Uint32Var(
+		&params.minNumValidators,
+		minNumValidators,
+		1,
+		"the minimum number of validators in the validator set for PoS.",
+	)
+	cmd.Flags().Uint32Var(
+		&params.maxNumValidators,
+		maxNumValidators,
+		math.MaxUint32,
+		"the maximum number of validators in the validator set for PoS.",
+	)
```

Comment:
Please remove the punctuation from the flag description 🙏

Reply:
fixed a91c866e0f6d742a0cc4af63f07196b4703ec96a